apps/api-reference/src/components/EvmApi/run-button.tsx

```diff
@@ -88,7 +88,18 @@ export const RunButton = <ParameterName extends string>(
       {status.type === StatusType.Results && (
         <div>
           <h3 className="mb-2 text-lg font-bold">Results</h3>
-          <Code language="json">{stringifyResponse(status.data)}</Code>
+          {props.type === EvmApiType.Write && status.data && typeof status.data === 'object' && 'hash' in status.data && typeof status.data.hash === 'string' ? (
+            <>
+              <Code language="json">{`Tx Hash: ${status.data.hash}`}</Code>
+              {'link' in status.data && typeof status.data.link === 'string' && (
+                <a href={status.data.link} target="_blank" rel="noopener noreferrer" className="text-sm text-blue-500 hover:underline">
```

Comment:
I'm thinking that it might be a better UI to just not show the tx hash in a `<Code>` at all; it's a bit strange to use a code block for something that isn't code or json.  Thoughts?

I also suggest using the `InlineLink` component instead of using `a` directly as the `InlineLink` component will apply styling that is consistent with links elsewhere in the app.

Finally, passing `rel="noopener noreferrer"` is [no longer needed](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener) as modern browsers will automatically apply `noopener` any link that has `target="_blank"` and `noreferrer` is not really all that meaningful to something like us (in fact it may be even more advantageous for us to expose that we're referring folks to other apps)